.buildkite/rllib_contrib.rayci.yml

```diff
@@ -0,0 +1,21 @@
+group: rllib contrib tests
+depends_on:
+  - oss-ci-base_build
+steps:
+  - label: ":exploding_death_star: rllib contrib: a2c tests"
+    if: build.env("BUILDKITE_PIPELINE_ID") == "0189942e-0876-4b8f-80a4-617f988ec59b"
+    instance_type: medium
+    tags: rllib_contrib
+    commands:
+      - conda deactivate
```

Comment:
could you change these into a bash script rather than encoding it on the pipeline?

Reply:
i'll probably leave rllib team's code untouched, CC: @sven1977 can you help with this as a follow up? thankks

Reply:
well or maybe i can just do it for them